src/com/janek/basics/math/MathUtil.java

```diff
@@ -58,4 +66,20 @@ public static double doCalculation(double one, String symbol, double two) {
                 throw new IllegalArgumentException("Unsupported symbol");
         }
     }
+
+    public static boolean isEven(int number) {
+        if (number % 2 == 0) {
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean isNumber(String text) {
```

Comment:
javadoc please

Reply:
The description should be:
```
**Checks** if the string is a number
```
and the return statement:
```
@return true if the string is a number, otherwise false
```